modules/cnv-edit-cdrom-vm.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-edit-vms.adoc
+// * cnv/cnv_users_guide/cnv-editing-vm-template.adoc
+
+// Establishing conditionals so content can be re-used for editing VMs
+// and VM templates.
+
+ifeval::["{context}" == "cnv-edit-vms"]
+:object: virtual machine
+:object-gui: Virtual Machines
+endif::[]
+
+ifeval::["{context}" == "cnv-editing-vm-template"]
+:object: virtual machine template
+:object-gui: Virtual Machine Templates
+endif::[]
+
+[id="cnv-vm-edit-cdrom_{context}"]
+
+= Editing CD-ROMs for virtual machines
+
+.Procedure
+
+. From the *{object-gui}* tab, select your {object}.
+. Select the *Overview* tab.
+. Click the pencil icon next to the *CD-ROMs* label to open the *Edit CD-ROM* window.
+* If no CD-ROMs are available for editing, the CD label initially displays *blank*.
+* If there are CD-ROMs avaiable, you can eject the CD-ROM by clicking *Eject CD-ROM* or remove it by clicking *-*.
+. In the Edit CD-ROM window, do the following:
```

Comment:
`Edit CD-ROM` should be `*Edit CD-ROM*` like in the earlier step.

Reply:
Done.